utility/path.go

```diff
@@ -67,3 +70,36 @@ func BaseFilter(base string, allowed bool) FilterFunc {
 		return (allowed == strings.EqualFold(base, b)), nil
 	}
 }
+
+// FindFileInAppDir ...
+func FindFileInAppDir(basePth, preferedAppName, fileName string) (string, error) {
```

Comment:
on osx an app extension is a dir simply, please merge the basepath and preferedAppName arguments as thos are going to be in the same path anyway, and expect a filepath with the suffix of .app

Reply:
Updated